### DISCUSSION TITLE
Support for Transducer Models

### DISCUSSION THREAD

**Discussion by agokrani** [PaddlePaddle/PaddleSpeech#2838] [Q&A]
Hi, 

First of all, great project. Very well managed code. Easy to understand and read. Do you guys plan to support transducer models for s2t?

**Comment by yt605155624** [comment]
RNN-T is in our development plan, do you have any more recommendations for the Transducer models?

**Reply by agokrani** [reply]
Why just RNN-T, if I look at the design of the repo, I think its very well suited to implement general neural transducer that can work with any type of encoder and predictor. If you already have a development branch with some code or design overview, I can also take a look and help in the development process
